superset/assets/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -576,11 +576,12 @@ export default class DateFilterControl extends React.Component {
     const endpoints = this.props.endpoints;
     value = value
       .split(SEPARATOR)
-      .map((v, idx) =>
-        moment(v).isValid()
-          ? v.replace('T00:00:00', '') +
-            (endpoints ? ` (${endpoints[idx]})` : '')
-          : v || (idx === 0 ? '-∞' : '∞'),
+      .map(
+        (v, idx) =>
+          (v.replace('T00:00:00', '') || (idx === 0 ? '-∞' : '∞')) +
+          (endpoints && value.includes(SEPARATOR)
```

Comment:
instead of using `value.includes` here, i think it's cleaner to use the 3rd argument for the callback in map and check if the length of the array is > 1.  see details here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map